parm/metplus/plot_latlon_points.py

```diff
@@ -0,0 +1,83 @@
+import argparse
```

Comment:
@mkavulich If this script is specifically for plotting only MET(plus) poly files, we should emphasize that by calling it something like `plot_met_poly_mask.py`.  Also it would be worth documenting this in the rst doc files so people know about it when creating new poly files (maybe you did but I missed it).